### DISCUSSION TITLE
Speed is slow, not reaching the target goal

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#1596] [Help with using Postal]
Hello community,

I am using 32 GB RAM Server with 960GB SSD | 15 IPs
----------------------------------------------------
Processor: E3-1230v2 (3.3Ghz Quad Core + HT)

vendor_id       : GenuineIntel
cpu family      : 6
model           : 58
model name      : Intel(R) Xeon(R) CPU E3-1230 V2 @ 3.30GHz
stepping        : 9
cpu MHz         : 1600.272
cache size      : 8192 KB
siblings        : 8
cpu cores       : 4

Below is the configuration 
-----------------------------------------
general:
  
  use_ip_pools: true
  maximum_delivery_attempts: 6
  use_local_ns_for_domains: true

main_db:
  
  host: 127.0.0.1
  username: postal
  password: p0stalpassw0rd
  database: postal
  pool_size: 20
  reconnect: true

message_db:
  
  host: 127.0.0.1
  username: postal
  password: p0stalpassw0rd
  prefix: postal
  pool_size: 20
  reconnect: true

workers:
  quantity: 20
  threads: 15

----------------------
Email Sending Speed only 

37.70 messages/minute

What else we can do to increase speed upto 500 messages/minute

**Comment by willpower232** [comment]
what is the output of `postal status` ?

**Reply by rccess** [reply]
![image](https://user-images.githubusercontent.com/32220165/133978645-3c420bb4-45cd-4bb0-b8a0-86e0c76c198d.png)
![image](https://user-images.githubusercontent.com/32220165/133978692-00289a38-ad10-44db-ba4e-890254f5de67.png)

**Reply by rccess** [reply]
@willpower232 what may be possible issue mate?

**Reply by willpower232** [reply]
it looks like everything is running where it should be, the only thing I note is that you've changed both threads and workers, is it better if you only change workers?

**Reply by rccess** [reply]
i have changed like this 

workers:
  quantity: 40
  threads: 5

but no improvement , Any idea what else we can try ?

**Reply by willpower232** [reply]
I guess the only other thing could be database settings, I think there were some old issues talking about what settings could be improved in MariaDB/MySQL.
